setup.py

```diff
@@ -80,6 +80,7 @@
         "pytest-instafail",
         # optional dependency of scipy to use scipy.datasets
         "pooch",
+        "ipympl",
```

Comment:
What's about putting it in `gui-jupyter` instead? Now `ipympl` is mature enough, works well with notebook and jupyterlab and will need to be used in the [notebook 7.0](https://jupyter-notebook.readthedocs.io/en/latest/migrate_to_notebook7.html) for which, the `nbaag` backend will not work anymore.